src/component.js

```diff
@@ -21,23 +21,23 @@ class NProgressContainer extends React.Component {
     const previousChangeStartCallback = Router.onRouteChangeStart;
     const previousChangeCompleteCallback = Router.onRouteChangeComplete;
     const previousChangeErrorCallback = Router.onRouteChangeError;
```

Comment:
Change this to use `router` from `props` instead of `Router`

Reply:
In my case, when using `router`, i got error `router` is `undefined`